interfaces/display_message.yaml

```diff
@@ -0,0 +1,43 @@
+description: >-
+  A module that implements this interface should be able to:
+  - store (add, remove, change) and retrieve predefined messages
+  - show messages on a display
+
+  When a display message contains a session id, the display message must be removed once the session has ended.
+cmds:
+  set_display_message:
```

Comment:
is `set` here like `display` a screen? If yes, then what is the meaning/usecase of get?

Reply:
ah its from the ocpp spec - maybe we can add this to the description of the module

Reply:
correct

we did not add that to the description because we also want to use it for internal messages.